cmd/operator/main.go

```diff
@@ -136,16 +138,22 @@ func main() {
 	cfg.DisableCompression = disableCompression
 
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
-		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
+		Scheme:             scheme,
+		MetricsBindAddress: metricsAddr,
+		WebhookServer: webhook.NewServer(webhook.Options{
+			Port: 9443,
+		}),
+		NewCache: func(config *rest.Config, opts ctrlcache.Options) (ctrlcache.Cache, error) {
+			opts.Namespaces = []string{namespace}
+
+			return ctrlcache.New(config, opts)
+		},
```

Comment:
Do we need to set this? Currently, we aren't setting it if I'm not wrong

Reply:
PTAL:

https://github.com/kedacore/keda/blob/1179eaa400ced857b8d1dc375a3f26800777bee0/cmd/operator/main.go#L138-L149

The `namespace` variable is provided by the following function:
https://github.com/kedacore/keda/blob/1179eaa400ced857b8d1dc375a3f26800777bee0/cmd/operator/main.go#L61-L69

I just replicated the same logic according to the new manager creation.

Reply:
I'd say more... Taking into account that `tls-min-version` isn't documented anywhere, I'd remove it directly as part of this commit in favor of the documented env. 
@zroubalik @tomkerkhove ?
Just to clarify, that parameter was introduced in v2.10 and is not documented anywhere, so if anyone is using it is because they have reviewed the code and used it under their own risk.